lib/routes/account.js

```diff
@@ -217,6 +217,7 @@ module.exports = function (
             .then(
               function (result) {
                 sessionToken = result
+                metricsContext.save(sessionToken, form.metricsContext, config)
```

Comment:
Good point. I've updated `lib/metrics/context.js` to use token-neutral language and am now saving metadata against the keyfetch token too. This means that the `account.keyfetch` activity event also has metadata, so the only event without metadata in this PR is `account.verified` now.